client/src/components/App.jsx

```diff
@@ -12,9 +12,13 @@ class App extends React.Component {
 		};
 	}
 
+	componentDidMount() {
+		this.getReviews();
+	}
+
 	getReviews() {
 		$.ajax({
-			url: '/',
+			url: '/reviews',
```

Comment:
@gorakong The spacing in this file, as well as other files in this commit, doesn't conform to Airbnb's style guidelines. Are you sure your linter is configured correctly?

Reply:
@jcelder I have yet to change the syntax of my files to comply to my linter/Airbnb's style guidelines, but I will make those changes now. Thanks for the reminder!